pkg/machinery/kernel/kernel.go

```diff
@@ -31,6 +31,14 @@ var DefaultArgs = []string{
 	"ima_template=ima-ng",
 	"ima_appraise=fix",
 	"ima_hash=sha512",
+	// TODO: configure it properly
+	"talos.selinux=permissive",
+	"audit=1",
```

Comment:
drop all these audit/debugging args from the defaults

Reply:
Yes, and also collecting support bundles in CI would be much better than https://github.com/siderolabs/talos/pull/9127/commits/0ac02dc9b6ea2ead19d4ef86648ce323e40f9464